internal/provider/model_provider.go

```diff
@@ -124,7 +123,7 @@ func (p ArgoCDProviderConfig) getApiClientOptions(ctx context.Context) (*apiclie
 			runtime.ErrorHandlers = runtimeErrorHandlers
 		}
 
-		err := headless.MaybeStartLocalServer(ctx, opts, "", nil, nil, cache.RedisCompressionNone, nil)
+		err := headless.MaybeStartLocalServer(ctx, opts, "", nil, nil, &clientcmd.DirectClientConfig{})
```

Comment:
https://github.com/argoproj/argo-cd/pull/21901/files#diff-170ef34ea6636f7c5461828e058b61f2e4e53f6f90d761e07be2fa1637d9eaf6
maybe `nil` instead of ` &clientcmd.DirectClientConfig{}`

Reply:
changed, now it's `nil`